osm2streets/src/transform/separate_cycletracks.rs

```diff
@@ -83,7 +83,7 @@ fn find_cycleways(streets: &StreetNetwork) -> Vec<Cycleway> {
                 {
                     cycleways.push(Cycleway {
                         cycleway: cycleway_road.id,
-                        cycleway_center: cycleway_road.untrimmed_road_geometry().0,
+                        cycleway_center: cycleway_road.center_line.clone(),
```

Comment:
This is fine here, and is the kind of thing we'll want to do almost everywhere. `untrimmed_road_geometry` should almost never be used once we maintain `center_line` as we go